userguide/tutorials/development.adoc

```diff
@@ -20,11 +20,16 @@ The first step is to clone the Kill Bill repository. For this, you need to follo
 
 . Click on *Clone a Git Repository and add the clone to this view*
 
-. In the Git Repository details, enter the Kill Bill Git URL https://github.com/killbill/killbill in the URI field. Enter your Git Credentials. Click *Next*
+. In the Git Repository details, enter the Kill Bill Git URL https://github.com/killbill/killbill in the *URI* field. 
+
+.. If you want read-write access to the Kill Bill repository, Select *https* in the *Protocol* dropdown and enter your Git credentials in the *User/Password* fields . 
+.. If you do not have Git credentials or want a read-only access, you can Select *Git* in the *Protocol* Dropdown. This disables the credentials fields. However, this will clone the repository anonymously and give you a read only access to the repository. 
```

Comment:
Didn't you have to set up an SSH key associated with your GitHub account as well? Or maybe you did it initially when you set up the killbill-docs repo? If I go to https://github.com/killbill/killbill while being logged-out, I don't see the Git protocol in the dropdown for instance.

Reply:
No, I did not do anything special for setting up SSH key.  If required, I can read up a bit more about this and see why/how it worked for me.